packages/magika.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,19 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolName = 'magika'
+    $category = 'File Information'
+    $executablePath = $toolName
+    $arguments = " -h"
+    $iconLocation = Join-Path ${Env:WinDir} "system32\cmd.exe" -Resolve
+
+    # Create output file to log python module installation details
+    $outputFile = VM-New-Install-Log ${Env:VM_COMMON_DIR}
```

Comment:
We need to keep the logs for all python libraries consistent. Either we should use the same file for all of them, or at least different files within the same location.

Reply:
I agree, but I can't seem to find a good way to programmatically get the same location, as it's located in `C:\ProgramData\chocolatey\lib\libraries.python3.vm\tools` and is based on the "Invocation Location": `$(Split-Path -parent $MyInvocation.MyCommand.Definition)`

We might need to adjust this location for `libraries.python3.vm` to be something else so that it's easier to find. I also think that the location that it ends up in is really weird, since most things `chocolatey` end up in `C:\ProgramData\chocolatey\logs`, and then we also use `C:\ProgramData\_VM` as a location for our logs.

Reply:
Added a PR to adjust `libraries.python3.vm` to use the same location as this package, which is more standardized.

https://github.com/mandiant/VM-Packages/pull/946